packages/opencensus-web-instrumentation-zone/src/interaction-tracker.ts

```diff
@@ -44,26 +49,38 @@ export class InteractionTracker {
       const time = Date.now();
 
       console.warn('Running task');
+      console.log(task);
       console.log(task.zone);
 
       let taskZone = this;
       if (isTrackedElement(task)) {
         console.log('Click detected');
 
-        const traceId = randomTraceId();
-        const tracingZone = Zone.root.fork({
-          name: traceId,
-          properties: {
-            isTracingZone: true,
-            traceId,
-          },
-        });
+        if (interactionTracker.shouldCreateNewZone) {
+          interactionTracker.shouldCreateNewZone = false;
+
+          // Timeout of 50 ms to reset flags to allow the creation of a new zone for a
+          // new user interaction.
+          setTimeout(() => {
```

Comment:
Could you explicitly run this in the root zone just in case this happens to be in a deeper zone? (E.g. the Angular zone when done in Angular)